src/pow.cpp

```diff
@@ -73,22 +73,3 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
 
     return bnNew.GetCompact();
 }
```

Comment:
then we should remove the undesired dependencies.

On Thu, Jul 14, 2016 at 4:30 PM, Nicolas Dorier <notifications@github.com>
wrote:

> In src/pow.cpp
> <https://github.com/bitcoin/bitcoin/pull/8339#discussion_r70815431>:
>
> > @@ -73,22 +73,3 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
> >
> >      return bnNew.GetCompact();
> >  }
>
> @jtimon <https://github.com/jtimon> actually, it is not ok, pow is not
> part of consensus (some methods are linked to CBlockIndex)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/8339/files/95b710aabc271222b669da505fc50abe67413df2..5eef3d92beecb72cef6dd6dd4af55ea15dc45b94#r70815431>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA9jSpRVFQNcuxXn4wPG4yGXQoihYfWbks5qVkfvgaJpZM4JMHBr>
> .
>